docs/pages/usage/namespace_offloading.md

```diff
@@ -0,0 +1,162 @@
+
+### Quick Offloading
+
+Since the very first version, Liqo provides a label-based mechanism to quickly set-up namespace offloading over remote clusters.
+
+To enable it, you should label a target namespace:
+
+```bash
+kubectl label namespace demo-namespace liqo.io/enabled=true
+```
+
+Pods scheduled in the "*demo-namespace*" will be potentially offloaded inside remote clusters. 
+With *quick offloading*, we are selecting all peered clusters suitable for offloading. 
+If you need a fine-grained approach, you can rely on custom offloading and the *namespaceOffloading* resource.
+
+### Custom Offloading
+
+As mentioned before, Liqo namespace enables the extension of the standard *v1.Namespace* over remote clusters. 
+To control all different aspects of the namespace extension, Liqo provides a *namespaceOffloading* resource. 
+The policies defined inside the *namespaceOffloading* specify how the local namespace can be replicated on peered clusters.
+
+Any namespace inside the home cluster can have its own *namespaceOffloading* resource and its corresponding deployment policy.
+
+In other words, the *namespaceOffloading* defines "*per-namespace boundaries*", limiting the scope where pods can be remotely offloaded.
+
+As presented in the following example, the *namespaceOffloading* resource is composed mainly by three fields:
+
+{{% render-code file="static/examples/namespace-offloading-default.yaml" language="yaml" %}}
+
+#### namespaceMappingStrategy
+
+The namespaceMappingStrategy defines the naming strategy used to create the remote namespaces. 
+The accepted values are:
+
+| Value               | Description |
+| --------------      | ----------- |
+| **EnforceSameName** | The remote namespaces have the same name as the namespace in the local cluster (this approach can lead to conflicts if a namespace with the same name already exists inside the selected remote clusters). |
+| **DefaultName** (Default)    | The remote namespaces have the name of the local namespace followed by the local cluster-id to guarantee the absence of conflicts. |
+
+{{% notice note %}}
+"(Default)" means that if you do not specify the field at the resource creation, it will be automatically generated with this default value.
+{{% /notice %}}
+
+{{% notice info %}}
+The **DefaultName** value is recommended if you do not have particular constraints related to the remote namespaces name. 
+However, using **DefaultName** policy, the namespace name cannot be longer than 63 characters according to the [RFC 1123](https://datatracker.ietf.org/doc/html/rfc1123). 
+Since adding the cluster-id requires **37 characters**, your namespace name can have at most **26 characters**.
+{{% /notice %}}
+
+#### podOffloadingStrategy
+
+The podOffloadingStrategy defines constraints only about pod scheduling. Standard k8s services are always replicated inside all the selected clusters.
```

Comment:
```suggestion
The podOffloadingStrategy defines constraints about pod scheduling. Standard Kubernetes services are always replicated inside all the selected clusters.
```

Might be appropriate to move the service part as a note below, to also better justify the reason for the Local policy.

Reply:
I have moved it, thank you